docs/_sidebar.md

```diff
@@ -13,13 +13,15 @@
     - [About merging pull requests](merging.md)
 - Reference
     - [Module authoring guidelines](implementation-guidelines.md)
-    - [Deployable Architecture authoring guidelines](da-implementation-guidelines.md)
+    - [DA authoring guidelines](da-implementation-guidelines.md)
```

Comment:
This was purposely shortened as its too long for side bar:
![image](https://github.com/user-attachments/assets/cf2d014e-438a-4c6e-a4fe-d311fc5f5eb5)